termqt/terminal_widget.py

```diff
@@ -129,7 +129,7 @@ def set_font(self, font: QFont = None):
             font = qfd.systemFont(QFontDatabase.FixedFont)
             info = QFontInfo(font)
 
-        font.setPointSize(12)
+        font.setPointSize(10)
```

Comment:
Same as above.

Reply:
The point-size I changed because it looked too big on my screen, will change it back, no problem.
Although, it would probably be better if it was settable.

Reply:
I agree. There are a lot of cosmetic settings scattered around and I wish there could be a config structure (like a dictionary or a class or a namedtuple?) that keeps all of these as an argument to the constructor.

You don't have to implement this idea if that's too much necessary work.